starter-code/scripts/views/articleView.js

```diff
@@ -86,6 +95,11 @@
    }; */
 
   // COMMENT: What does this method do?  What is it's execution path?
+  //this method instantiates everything! It goes over the allArticles object
+  //array populates all the information on the main home page, from there it hides
+  //the content of the articles if they are greater then a certain length. While this
+  //is happening it also calls populateFilters and handleFilters which ensures no
+  //duplicates are displayed and populates the navigation options.
```

Comment:
This method is kind of the meat of our app! It is called in maaaaany of our routes after we've attached the appropriate articles to our ctx object. 

This method first shows the articles section and hides all other primary sections (such as the about-me and admin sections). It empties out the articles section of all articles and takes the articles passed in (many times from the ctx.articles property, depending on where this method is being called) and appends them to the page after rendering them (which compiles html template, adds some properties, and returns the compiled template). It then calls articleView.populateFilters and articleView.handleFilters to append authors and categories to the appropriate select boxes and attach an event listener to those select boxes. It then truncates articles if they are longer than 1 paragraph.